includes/internal/emails/views/html-settings.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * File containing view for the tools page.
+ *
+ * @package sensei
+ * @since $$next-version$$
+ *
+ * @var array              $options Array of Sensei settings and corresponding values.
+ * @var array              $fields_to_display Array of Email settings to display.
+ * @var Email_Settings_Tab $this Email_Settings_Tab instance.
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+$fields_to_display = isset( $fields_to_display ) ? $fields_to_display : [];
+$options           = isset( $options ) ? $options : [];
+?>
+<form id="email-notification-settings-form" action="options.php" method="post">
+	<?php settings_fields( 'sensei-settings' ); ?>
+	<?php
+	foreach ( $options as $option_name => $option_data ) {
+		// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
```

Comment:
Just curious, didn't see any issue locally after removing it, does it show as an issue in the CI?

Reply:
Yes, was getting errors in CI, here is an example: https://github.com/Automattic/sensei/actions/runs/4169962579/jobs/7218469829